node/network/statement-distribution/src/lib.rs

```diff
@@ -493,27 +500,6 @@ fn check_statement_signature(
 		.and_then(|v| statement.check_signature(&signing_context, v))
 }
 
-type StatementListeners = Vec<mpsc::Sender<SignedFullStatement>>;
```

Comment:
Why did you remove this?!

Please bring it back. That is required by cumulus.

Reply:
Will reinstate - misunderstood the origin of this code and thought it was unused.